src/app/theme/colorScheme.ts

```diff
@@ -0,0 +1,12 @@
+import { Appearance, ColorSchemeName } from 'react-native'
+
+import flag from 'cozy-flags'
+
+export const getColorScheme = (): ColorSchemeName => {
+  if (flag('ui.darkmode.enabled')) {
```

Comment:
Will this works on welcome screen as we are not connected to any cozy-client here?

Reply:
Updated! See last commit.